src/sentry/static/sentry/app/data/forms/inboundFilters.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {t, tct} from '../../locale';
+import ExternalLink from '../../components/externalLink';
+
+// Export route to make these forms searchable by label/help
+export const route = '/settings/organization/:orgId/project/:projectId/filters/';
+
+const newLine = t('Separate multiple entries with a newline. ');
```

Comment:
Nit: I feel like `newLine` has some built-in connotations (e.g. a const for `\n` or something), so I'd probably name as `newLineHelpText` and `newLineAndGlobHelpText`.

(Also, is the trailing space intentional?)